tests/helper/config.go

```diff
@@ -0,0 +1,92 @@
+package helper
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+var KEDATestConfig = TestConfig{
+	KEDA: &KEDAConfig{
+		InstallKeda:   true,
+		InstallKafka:  true,
```

Comment:
Should we include here all the modifiers like cloud pod identities?

Reply:
Not really sure, I don't want to necessarily include every single configuration because then this would be sort of like re-implementing the use case of an `.env` file. 

If people have good reasons for purposefully including an option (e.g., if many users require it) then it would make sense?

Reply:
agree, let's add later if needed